Havit.Blazor.Components.Web.Bootstrap/TreeViews/ValueWrapper.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Havit.Blazor.Components.Web.Bootstrap
+{
+	public class ValueWrapper<TValue> : INotifyPropertyChanged
```

Comment:
@sshulik,
Do we need the `ValueWrapper<TValue>` class et all? From my point of view it just extracts logic which should sit directly in `HxTreeViewItem`. Why don't we simplify the whole logic:

```razor ddd
@foreach (var item in ChildrenSelector(this.Value))
{
   <HxTreeViewItem
       Level="this.Level + 1"
       Value="item"
       ChildrenSelector="this.ChildrenSelector"
       ...
}
```
...instead of creating a parallel structure representing the tree node?

Reply:
Refactored to Blazor-native concept.